backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -221,6 +223,7 @@ public Parcel Update(Parcel parcel)
                     }
                     else
                     {
+                        CanUpdateOrRemoveProperty(existingBuilding.ProjectNumber, isAdmin);
```

Comment:
The current way the function is created it will attempt to update all buildings within a parcel.  Which would result in throwing this error, even if they didn't make any changes to the specific building.  This would be very frustrating to a user to get this error when they did not edit the building in question.

Reply:
Additionally it would appear you haven't ensured they can't delete the building lower down within this Update function.